.docs/config.md

```diff
@@ -124,18 +130,30 @@ This section contains all the different configuration related to the PostgreSQL
 | `max_open_connections` | `integer` | Max number of open connections at any time (default: `1`) | `15` | 
 
 ## `pruning`
-This section contains the configuration about the pruning options of the database. Note that this will have effect only if you add the `"pruning"` entry to the `modules` field of the [`cosmos` config](#cosmos). 
+This section contains the configuration about the pruning options of the database. Note that this will have effect only
+if you add the `"pruning"` entry to the `modules` field of the [`cosmos` config](#cosmos).
 
 | Attribute | Type | Description | Example |
 | :-------: | :---: | :--------- | :------ |
 | `interval` | `integer` | Number of blocks that should pass between one pruning and the other (default: prune every `10` blocks) | `100` | 
 | `keep_every` | `integer` | Keep the state every `nth` block, even if it should have been pruned | `500` | 
 | `keep_recent` | `integer` | Do not prune this amount of recent states | `100` |
 
-## `logging` 
-This section allows to configure the logging details of BDJuno. 
+## `logging`
+This section allows to configure the logging details of Juno.
 
 | Attribute | Type | Description | Example |
 | :-------: | :---: | :--------- | :------ |
 | `format` | `string` | Format in which the logs should be output (either `json` or `text`) | `json` | 
 | `level` | `string` | Level of the log (either `verbose`, `debug`, `info`, `warn` or `error`) | `error` | 
+
+## `telemetry`
+This section allows to configure the telemetry details of Juno.
+
+| Attribute | Type | Description | Example |
+| :-------: | :---: | :--------- | :------ |
+| `enabled` | `bool` | Whether the telemetry should be enabled or not | `false` | 
+| `port` | `uint` | Port on which the telemetry server will listen | `8000` | 
+
+**Note**  
+If the telemetry server is enabled, a new endpoint at the provided port and path `/prometheus` will expose [Prometheus](https://prometheus.io/) data.
```

Comment:
The convention is to use "/metrics" - that's the default in prometheus, it can be configured as something else but better to stick to the convention here